File: Python/Week One/THISFUNCTION.py
class FBTeam(object):
    def __init__(self, name, city, color):
        self.name = name
        self.city = city
        self.color = color
        self.players = []
    def __str__(self):
        return self.name + ", " + self.city + ", " + self.color + ", " + str(self.players)

class Player(object):
    def __init__(self, name, position, number):
        self.name = name
        self.position = position
        self.number = number
    def __str__(self):
        return self.name + " " + self.position + " " + self.number

USC = FBTeam("Trojans", "LA", "Red, Yellow")
USC.players.append(Player("Reggie Bush", "Running Back", "5"))
USC.players.append(Player("Reggie Bush", "Running Back", "5"))
for player in USC.players:
    print player
print Player("Reggie Bush", "Running Back", "5")
print USC

File: Python/Week One/THISFUNCTION.js
function FBTeam(name, city, color) {
  this.name = name;
  this.city = city;
  this.color = color;
  this.players = [];
}

function player(name, position, number) {
  this.name = name;
  this.position = position;
  this.number = number;
}

var USC = new FBTeam("Trojans", "LA", "Red, Yellow");
USC.players.push(new player("Reggie Bush", "Running Back", "5"));

var SH = new FBTeam("Seahawks", "Seattle", "Blue, White, Green");
SH.players.push(new player("Richard Sherman", "Cornerback", "25"))


console.log(USC, SH);
